src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -588,7 +588,96 @@ The next four parameters are optional, and apply to advanced Azure endpoint conf
+- *windowsInternalCustomStartupScriptUrl:* If you are using Windows, provide a URL of a valid Windows PowerShell script to configure your nodes. We strongly recommend to use the default provided script. You are free to copy the script and add custom steps, but be aware that any modification to the default script might cause the deployment to fail. We recommend to use _userCustomStartupScriptUrl_ field to add your custom configurations.
+- *userCustomStartupScriptUrl:* Optional parameter. Use it to provide a URL of a valid custom script (_.sh_ for Linux, _.ps1_ for Windows) that will be called at the VMs' startup. The script is called after the default ProActive configuration and before the node application is actually executed. You can use this script, for instance, to create directories, assign permissions, or download utilities.
++
+[NOTE]
+====
+Note that this user custom script will be run as root/admin user.
+====
+- *nodePreCommand:* Optional parameter. This multi-line field can be used to provide specific commands to execute right before launching the ProActive's node.jar application. The main difference with the previous user custom script is that _nodePreCommand_ will be run as _the same_ user as node.jar. This field is thus useful for operations that require matching users, e.g. mounting a shared volume in Windows, or creating non-root paths/files in Linux. Please note the OS-specific conditions for these commands:
+* *_Linux:_* The commands will be appended as `ExecStartPre` lines to a _.service_ file. Therefore, commands should include their full path, e.g. `mkdir` should be written as `/bin/mkdir` or `chown` as `/bin/chown`.
+* *_Windows:_* The commands will be included in a _.bat_ batch file and must therefore be DOS-compatible. As a guideline, here is an unofficial https://www.robvanderwoude.com/batchcommands.php[list of DOS commands] and a http://steve-jansen.github.io/guides/windows-batch-scripting/part-1-getting-started.html[scripting tutorial].
+- *jvmParameters:* Add JVM parameters to configure your ProActive node, use the format _-Dproperty=value_. The default values allow for a correct deployment of the node. You can add your own parameters but be aware that removing the current values might cause the deployment to fail. A non-exhaustive list of node-related JVM parameters can be found in your configuration file `$PROACTIVE_HOME/config/network/node.ini`
+- *armTemplateUrl:* Optional parameter, for advanced configuration only. Provide a URL with a valid _.json_ Azure Resource Management https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-authoring-templates[configuration file]. If set, this file will override the default ARM file.
```

Comment:
The ARM file is a windows file that enable to set the group authorization ? If it is not the case please add explanation about the ARM file.

Reply:
> The ARM file is a windows file that enable to set the group authorization ?
I don't understand from where did you infer this. The ARM templates are a whole topic, that's why I put a link to the documentation

Reply:
Fixed it, let me know if it's clearer @MaelAudren